src/clib/lib/job_queue/job_kw_definitions.cpp

```diff
@@ -23,3 +25,32 @@ config_item_types job_kw_get_type(const char *arg_type) {
 
     return type;
 }
+
+const char *kw_from_type(config_item_types typ) {
+    switch (typ) {
+    case CONFIG_STRING:
+    case CONFIG_PATH:
+    case CONFIG_EXISTING_PATH:
+    case CONFIG_EXECUTABLE:
+        return JOB_STRING_TYPE;
```

Comment:
I believe for consistency with the rest of the code base, this should look as follows:

```
    case CONFIG_STRING:
        return JOB_STRING_TYPE;
    case CONFIG_PATH:
        return JOB_STRING_TYPE;
    case CONFIG_EXISTING_PATH:
        return JOB_STRING_TYPE;
    case CONFIG_EXECUTABLE:
        return JOB_STRING_TYPE;
```

Reply:
Thing is, clang-tidy complains about that.

Reply:
sorry, apparently an unsuccessful attempt at a joke